source/include/sigv4_internal.h

```diff
@@ -83,39 +83,42 @@
  * @brief Default value when HttpParameters_t.pPath == NULL.
  */
 #define HTTP_EMPTY_PATH                        "/"
-#define HTTP_EMPTY_PATH_LEN                    ( sizeof( HTTP_EMPTY_PATH ) - 1U )               /**< The length of #HTTP_EMPTY_PATH. */
+#define HTTP_EMPTY_PATH_LEN                    ( sizeof( HTTP_EMPTY_PATH ) - 1U )                 /**< The length of #HTTP_EMPTY_PATH. */
 
-#define URI_ENCODED_SPECIAL_CHAR_SIZE          3U                                               /**< The size of an encoded URI special character. */
-#define URI_DOUBLE_ENCODED_EQUALS_CHAR_SIZE    5U                                               /**< The size of the double-encoded "=" character. */
+#define URI_ENCODED_SPECIAL_CHAR_SIZE          3U                                                 /**< The size of an encoded URI special character. */
+#define URI_DOUBLE_ENCODED_EQUALS_CHAR_SIZE    5U                                                 /**< The size of the double-encoded "=" character. */
 
-#define LINEFEED_CHAR                          '\n'                                             /**< A linefeed character used to build the canonical request. */
-#define LINEFEED_CHAR_LEN                      1U                                               /**< The length of #LINEFEED_CHAR. */
+#define LINEFEED_CHAR                          '\n'                                               /**< A linefeed character used to build the canonical request. */
+#define LINEFEED_CHAR_LEN                      1U                                                 /**< The length of #LINEFEED_CHAR. */
 
-#define SPACE_CHAR                             ' '                                              /**< A linefeed character used to build the Authorization header value. */
-#define SPACE_CHAR_LEN                         1U                                               /**< The length of #SPACE_CHAR. */
+#define CARRIAGE_RETURN_LINEFEED_STRING        "\r\n"                                             /**< The string used in non-canonicalized HTTP headers to separate header entries in HTTP request. */
+#define CARRIAGE_RETURN_LINEFEED_STRING_LEN    ( sizeof( CARRIAGE_RETURN_LINEFEED_STRING ) - 1U ) /**< The length of #CARRIAGE_RETURN_LINEFEED_STRING. */
```

Comment:
The first 31 characters of these two macros are identical, which is a MISRA 5.4 violation

Reply:
Renamed the macros